src/switchroot/ostree-mount-util.h

```diff
@@ -153,15 +149,15 @@ get_aboot_root_slot (void)
 static inline char *
 get_ostree_target (void)
 {
-  autofree char *ostree_cmdline = read_proc_cmdline_key ("ostree");
-
-  if (!ostree_cmdline)
-    return NULL;
+  autofree char *slot_suffix = read_proc_cmdline_key ("androidboot.slot_suffix");
+  if (slot_suffix)
+    return get_aboot_root_slot (slot_suffix);
 
-  if (strcmp (ostree_cmdline, "aboot") == 0)
-    return get_aboot_root_slot ();
+  autofree char *ostree_cmdline = read_proc_cmdline_key ("ostree");
+  if (ostree_cmdline)
+    return steal_pointer (&ostree_cmdline);
```

Comment:
I think this is just `return read_proc_cmdline_key ("ostree")` now.

Reply:
Ah yes it is, thanks